libs/sdk-bindings/bindings-android/lib/src/main/kotlin/breez_sdk_notification/job/LnurlPayInfo.kt

```diff
@@ -58,7 +69,7 @@ class LnurlPayInfoJob(
             val response =
                 LnurlPayInfoResponse(
                     request.callbackURL,
-                    nodeState.inboundLiquidityMsats,
+                    maxSendable,
                     1000UL,
```

Comment:
Maybe the minSendable should be the ofpResp.minMsat if there is no inbound available, as it has to cover the min channel opening fee. What do you think?

Reply:
Yes, it makes sense.

Reply:
Done.
[Use OpeningFeeParams.minMsat for the minSendable](https://github.com/breez/breez-sdk/pull/833/commits/e6c29df8576b265dfe7ecbea1a7d0e52fba89368)

Reply:
But only if there is no inbound (no channels) and a channel is needed, else the minSendable is always something like 2_500_000, whereas if we have inbound minSendable should be 1_000

Reply:
Ah, I see. Sorry, been jumping between a lot of tasks today.

Considering `fee_msat` is 0 if no channel is needed:
```rust
/// Opening fee for receiving the amount set in the [OpenChannelFeeRequest], in case it was set.
/// It may be zero if no new channel needs to be opened.
pub fee_msat: Option<u64>,
```
It should be
```kotlin
val minMsat = ofpResp.feeMsat?.let { if (it == 0UL) 1000UL else ofpResp.feeParams.minMsat }!!
if(request.amount < minMsat ...
```

Reply:
Done f9f6ab7b791ff66b7046d9e7a6b8b8fcd1972ad2